build/common.sh

```diff
@@ -573,23 +542,47 @@ function kube::build::ensure_data_container() {
     # We want this to run as root to be able to chown, so non-root users can
     # later use the result as a data container.  This run both creates the data
     # container and chowns the GOPATH.
+    #
+    # The data container creates volumes for all of the directories that store
```

Comment:
Ahh.  Yes, well, it would be GREAT if we didn't need the data container.  Eventually, maybe?